### ISSUE TITLE
Train and test data

### ISSUE THREAD

**Issue by billhhh** [carrenD/ummkd#8]
Hi, thanks for sharing your good work and I am trying to follow your work.
I have questions regarding to your training and testing data. Really appreciate you could answer them:

1. In the original MMWHS Challenge, there are 20 ct and 20 mri. May I know which IDs of ct and mri are for model training? and which IDs of ct and mri for model validation?
2. Is there a testing set? (I am asking this since in your Medical-Cross-Modality-Domain-Adaptation repo, there is only ct testing set)

Much appreciate that you could help!

**Comment by chke097** [comment]
@billhhh      Hello, have you solved this problem？
I don't know which part is used for training set and validation set, especially how to choose the validation set.
Looking forward to your reply.
Best wishes to you.
